components/assignmentList.js

```diff
@@ -70,16 +70,15 @@ const AssignmentsList = ({
   // Group the assigned drills by date
   const groupedData = useMemo(() => {
     return assigned_data.reduce((acc, curr) => {
-      //round the assigned time to the nearest day
-      const roundedTime = Math.floor(curr.assignedTime / 86400000) * 86400000;
+      const time = getTimezoneOffsetTime(curr.assignedTime);
```

Comment:
This should be converted here because of date grouping, but today also needs to be converted